core/injector/application_injector.cpp

```diff
@@ -765,7 +765,8 @@ namespace {
         libp2p::injector::makeHostInjector(
             libp2p::injector::useWssPem(config->nodeWssPem()),
             libp2p::injector::useSecurityAdaptors<
-                libp2p::security::Noise>()[di::override]),
+                libp2p::security::Noise,
+                libp2p::security::Plaintext>()[di::override]),
```

Comment:
👀 Pretty sure this was not meant to be in a production code

Reply:
`Plaintext` was added. Useful for testing, tools, and alternative clients

Reply:
cc @kamilsa

Reply:
I agree with Igor. It is better to include this protocol when needed, not to have it in production. In future when we will work on SASSAFRAS privacy of messaging will matter more